docs/auxil/admonition_inserter.py

```diff
@@ -194,13 +196,16 @@ def _create_available_in(self) -> dict[type, str]:
                     if "telegram" not in name_of_class_in_prop:
                         continue
 
-                    # Writing to dictionary: matching the class found in the docstring to the
-                    # property of the class being inspected.
-                    # The class in the property docstring is the key, and the property of the
-                    # class currently being inspected is the value.
-                    attrs_for_class[self._resolve_class(name_of_class_in_prop)].add(
-                        f":attr:`{name_of_inspected_class_in_docstr}.{prop_name}`"
-                    )
+                    # Writing to dictionary: matching the class found in the docstring and its
+                    # subclasses to the property of the class being inspected.
+                    # The class in the property docstring (or its subclass) is the key,
+                    # and the property of the class currently being inspected is the value.
+                    cls = self._resolve_class(name_of_class_in_prop)
```

Comment:
shouldn't we check for `None` here? That's what was done in the attribute section (`if resolved_class is None ..`)